src/software/logger/protobuf_sink.cpp

```diff
@@ -76,15 +76,7 @@ void ProtobufSink::sendProtobuf(g3::LogMessageMover log_entry)
 
 std::ostream& operator<<(std::ostream& os, const google::protobuf::Message& message)
 {
-    // Pack into Any
-    google::protobuf::Any any;
-    any.PackFrom(message);
-
-    // Serialize into any
-    std::string serialized_any;
-    any.SerializeToString(&serialized_any);
-
     os << TYPE_DELIMITER << message.GetTypeName() << TYPE_DELIMITER
-       << base64_encode(serialized_any);
+       << base64_encode(message.SerializeAsString());
```

Comment:
Above this could you add a comment explaining why we do base64 encoding? Something like:

> We have to base64 encode the serialized protobuf before sending it to g3log since g3log treats \n characters as a new log msg which breaks the serialized protobuf.

Reply:
done